cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -68,7 +68,9 @@ def _curie_remove_suffix(term_id: str, suffix_def: dict) -> Tuple[str, str]:
                 search_results = re.search(r"%s$" % suffix, term_id)
                 if search_results:
                     stripped_term_id = re.sub(r"%s$" % suffix, "", term_id)
-                    if ONTOLOGY_CHECKER.is_valid_term_id(ontology_name, stripped_term_id):
+                    if ONTOLOGY_CHECKER.is_valid_term_id(
```

Comment:
Is this line too long for the formatting rules? If not can you revert this formatting change?

Reply:
Most of these little changes were done automatically by lint or fmt, which wasn't run in a previous PR